.github/workflows/create-release-api.yml

```diff
@@ -1,10 +1,12 @@
-name: "Create Release For Lifecycle Manager API Module"
+name: "Create Release For KLM API Module"
```

Comment:
To Reviewer: shortered the name so that more distinct between API and Image in the action page.

https://github.com/kyma-project/lifecycle-manager/actions